src/main/java/com/mycompany/items/Pumphandle.java

```diff
@@ -24,7 +24,7 @@ public boolean use() {
         }
 
         game.getCurrentPointOfInterest().setFixed();
-        game.getCurrentRoom().getPointOfInterest(PointsOfInterest.BOY.getName()).setDescription("Thank you for fixing the pump, I will never play with water again!");
+        game.getCurrentRoom().getPointOfInterest(PointsOfInterest.BOY.getName()).setDescription("“Thank you for fixing the pump, I will never play with water again!“");
```

Comment:
Hvis du er prøvede på at have `"` inde i en streng, så skriv `\"` i stedet for.

Reply:
Tak, skal jeg huske fremover 😄